Revoke-Obfuscation.psm1

```diff
@@ -477,100 +502,125 @@ http://www.leeholmes.com/blog/
72.5484, 45222.4953, 995.4257, -1847.5000, -1251.8610, -13419.0000, -2025.0848, -72045.0324, -4015.7678, -4019.5024, -4019.5024, -4006.7970, -4019.5024, -12.7054, 8013.1118, 18975.0105, 7941.5842, 9924.5685, 1706.5085, 9050.4421, -2923.7944, 1165.9527, -34851.5170, 28214.4857, 16576.4687, -27048.5330, 15120.7354, -5406.0535, -14477.6501, -3813.4209, -2661.4471, -4766.1196, -11816.2031, -37185.0000, -8437.3588, -40884.5000, -6794.9032, -48941.4964, 170618.1916, -110440.4976, -13044.4818, -24304.5000, -2358.5792, 1664.0000, -1769.4345, -36565.5000, -10533.7847, -43317.0000, -12477.3564, 109813.0000, 1374.8953, -61734.2500, 409.3292, -11032.2500, -71825.6653, -122693.7500, -28759.7479, -204302.2500, -14464.1837, 77562.0000, -2500.0088, 39977.5000, -1303.4766, 77616.5000, -1488.8287, -59264.5000, -4860.2913, -139886.5000, -1540.7090, -59799.0000, -3817.4242, 22734.5000, 1611.8041, -160326.0000, -12369.5944, 3332.2500, -17571.2553, 165476.0000, 24423.9924, -7328.5000, -5952.9037, 316946.7500, -6035.5864, 130329.0000, -9298.4555, 317753.7500, -13545.0009, -40698.0000, 9288.3108, -33731.5000, -20222.6042, 139661.2501, 34361.3069, -168896.7500, 84301.0371, 229835.7501, 35410.5494, -170293.5000, -41054.0702, -81213.4988, -8853.2501, -94670.4988, -41823.0957, 15002.5000, -7219.7348, -8812.5000, 1820.4115, 47851.5000, 19667.8130, -11568.0000, -22348.3947, -17160.5000, -6318.2706, 25354.0000, -818.3695, -62130.0000, -2464.7560, -3392.5000, 4397.0757, -52732.4976, -186823.4881, 0.0000, 0.0000, -49492.0000, 8260.7376, 127702.7500, -58360.9455, 0.0000, 0.0000, 0.0000, 0.0000, 80618.2500, 31231.8460, 272379.2471, 96801.4454, 0.0000, 0.0000, 164627.5000, -66189.7195, 94813.2539, -2228.0598, 52383.2500, -48763.2717, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, -224750.5000, -36484.9437, 291425.2531, -47532.4768, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, -131.0000, -60.9302, 67450.0000, -22906.3276, 101053.7496, 14247.7563, 0.0000, 0.0000, 0.0000, 0.0000, 71326.5000, -23759.0491, -195406.0000, -72403.8205, 0.0000, 0.0000, 36708.7500, -20507.9635, 122147.5000, -52614.2131, -30274.7461, -38591.6714, 280959.7535, -8396.3696, -889.0000, 691.0561, 43752.0000, -8189.7425, -81321.5000, 7613.5143, -58647.2500, -3625.1276, 22186.0000, -2961.3794, 150150.7496, 29831.3633, 329531.0000, -52370.5268, -92096.0000, -964.9300, -14362.7476, -88474.6270, -75909.0000, -22627.0451, 42606.7496, 31713.4313, 0.0000, 0.0000, 0.0000, 0.0000, 18990.0000, -10369.0641, 85862.9971, 30202.7383, 40866.5000, 4227.4575, -42388.2500, -104092.9893, 32168.5000, 132.9601, -54184.7500, -3552.2871, -92069.5000, -28045.6090, -34491.0029, 3344.8714, 0.0000, 0.0000, -89281.5000, -3081.1435, 16015.5079, -108361.1943, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, -58143.4950, 240250.8165, -19287.5000, -2909.9336, -8637.5000, -1257.2493, 0.0000, 0.0000, 0.0000, 0.0000, -122187.5000, -1826.0633, 979.0000, -522.9380, -652
```

Comment:
if you have a terminating error (no point carrying on) use throw instead of Write-Error